### PR TITLE
Remove all PyPi classifiers as this package is not published to PyPi

### PR DESCRIPTION
**Pull request by ChrisCarini**
## Proposed change

- Remove all PyPi classifiers as this package is not published to PyPi

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (which adds functionality to the supervisor)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:
- Link to cli pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast supervisor tests`)
- [ ] Tests have been added to verify that the new code works.

If API endpoints of add-on configuration are added/changed:

- [ ] Documentation added/updated for [developers.home-assistant.io][docs-repository]

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[docs-repository]: https://github.com/home-assistant/developers.home-assistant

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
         "Topic :: Scientific/Engineering :: Atmospheric Science",
         "Development Status :: 5 - Production/Stable",
         "Intended Audience :: Developers",
-        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.11",
```
Comment (frenck) [inline setup.py:28]: This is not a published package; I suggest to remove all classifiers at is has no function.

Comment (ChrisCarini) [reply setup.py]: @frenck - just removed them all; let me know if you had something different in mind here.

**Review by mdegat01 — CHANGES_REQUESTED** [review]

File: setup.py [hunk 2]
```diff
@@ -15,18 +15,7 @@
...
-        "License :: OSI Approved :: Apache Software License",
-        "Operating System :: OS Independent",
-        "Topic :: Home Automation",
-        "Topic :: Software Development :: Libraries :: Python Modules",
-        "Topic :: Scientific/Engineering :: Atmospheric Science",
-        "Development Status :: 5 - Production/Stable",
-        "Intended Audience :: Developers",
-        "Programming Language :: Python :: 3.8",
-    ],
+    classifiers=[],
```
[the remark this answers is not in the document]
Comment (ChrisCarini) [reply setup.py]: @mdegat01 nice! For some reason I missed that when I looked; thank you!

**Review by mdegat01 — APPROVED** [review]

**Comment by mdegat01** [comment]
Thanks @ChrisCarini 👍

**Pull request merged**